src/main/java/com/cburch/logisim/fpga/hdlgenerator/HdlParameters.java

```diff
@@ -32,7 +32,7 @@ public class HdlParameters {
   public static final int MAP_LN2 = 5;
   public static final int MAP_INT_ATTRIBUTE = 6;
   public static final int MAP_GATE_INPUT_BUBLE = 7;
-  public static final int MAP_POW2 = 8;
+  public static final int MAP_2POW = 8;
```

Comment:
Personally, I would leave the old name `MAP_POW2`, which is in line with `MAP_LN2`, i.e. the base is added as a suffix. Additionally, the change is thus kept minimal. Thus only line 166 needs to be changed to fix issue #1690.